Automattic/wp-calypso
# Change skeleton animation to comply with dotcom

p1689617587051499-slack-C053FAERCDP

## Proposed Changes

* Change skeleton animation color to `#F6F7F7` -> `#EEEEEE`

## Testing Instructions

- Checkout this branch in your local environment
- Build assets (`yarn && yarn start`)
- Open **Tools > Advertising**
- Then open the following in separate tabs: Posts, Campaigns, and Campaign Details (we need to make sure they will be ready to use before forcing the infinite loading state)
- Put `true` to force loading state to:
  - [main.tsx:94](https://github.com/Automattic/wp-calypso/blob/trunk/client/my-sites/promote-post-i2/main.tsx#L94),
  - [main.tsx:134](https://github.com/Automattic/wp-calypso/blob/trunk/client/my-sites/promote-post-i2/main.tsx#L134), and
  - [components/campaign-item-page/index.tsx:44](https://github.com/Automattic/wp-calypso/blob/trunk/client/my-sites/promote-post-i2/components/campaign-item-page/index.tsx#L44)
- Refresh those views and make sure that the animation range is the one Suggested by Bart

Some samples:

https://github.com/Automattic/wp-calypso/assets/115007291/255931cc-cd03-46de-a3e6-a3f5a2a1307a

(checking that the colors are correct ones)

https://github.com/Automattic/wp-calypso/assets/115007291/d4b21cb7-1421-4f14-8a36-319324ec8d27

(checking what updated colors look like)



## Pre-merge Checklist
